operator/roles/kiali-deploy/templates/openshift/console-links.yaml

```
@@ -0,0 +1,30 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleLink
+metadata:
+  name: kiali-app-link
+  namespace: {{ kiali_vars.deployment.namespace }}
+  labels:
+    app: kiali
+    version: {{ kiali_vars.deployment.version_label }}
+spec:
+  href: {{ kiali_route_url }}
+  location: ApplicationMenu
+  text: Kiali
+{% for namespace in namespaces %}
+---
+apiVersion: console.openshift.io/v1
+kind: ConsoleLink
+metadata:
+  name: "kiali-namespace-{{ namespace }}-link"
```

Let's get the names of the namespace links consistent with the app link name. As proposed earlier, the app name link will be something like `"kiali-app-link-{{kiali_vars.deployment.namespace}}"` so let's call the namespace links in the same form only instead of `kiali-app` call it `kiali-namespace` so this:

```suggestion
  name: "kiali-namespace-link-{{ namespace }}"
```

If we want to make these links easily associated with the kiali app they belong to, we could put the deployment namespace in here as well, like this:

```
  name: "kiali-namespace-link-{{ kiali_vars.deployment.namespace }}-{{ namespace }}"
```